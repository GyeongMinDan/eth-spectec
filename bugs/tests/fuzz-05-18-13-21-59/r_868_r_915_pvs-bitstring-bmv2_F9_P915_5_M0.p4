extern packet_in {
}
          struct standard_metadata_t {
}
          struct my_packet {
}
          struct my_metadata {
}
          parser MyParser(packet_in b, out my_packet p, inout my_metadata m, inout standard_metadata_t s) {
          state start {             transition b;           }
        }
