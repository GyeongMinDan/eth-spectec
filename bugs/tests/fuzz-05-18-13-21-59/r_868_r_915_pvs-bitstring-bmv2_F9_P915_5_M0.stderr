likely_hits/r_868_r_915_pvs-bitstring-bmv2_F9_P915_5_M0.p4(9): [--Wwarn=unused] warning: Parser MyParser is not used; removing
          parser MyParser(packet_in b, out my_packet p, inout my_metadata m, inout standard_metad...
                 ^^^^^^^^
[--Wwarn=missing] warning: Program does not contain a `main' module
